### PR TITLE
(feat): Add atd_offset to posfilter config model

### PR DESCRIPTION
**Pull request by madhavmk**
* Linked to Issue #211 
* Add AtdOffset pydantic model to handle antenna transducer offset config.

### CODE REVIEW

**Comment by madhavmk** [comment]
Hello @lsetiawan,
Before merging this PR, I need your help in adding valid sample test data to the tests/data/2022/NCL1/208/posfilter/RPH file.
Thanks!

File: tests/data/2022/NCL1/208/posfilter/RPH
Comment (lsetiawan) [inline tests/data/2022/NCL1/208/posfilter/RPH:1]: Please remove this file

**Review by lsetiawan — APPROVED** [review]
Update the commented stuff above, otherwise you are good to merge when ready 😄

**Pull request merged**
